### PR TITLE
Correct grammar & spelling for internships page

### LINKED ISSUE #459: Correct the spelling and grammar errors of the entire docs website.
**Describe the bug**
Currently the docs website has a lot of spelling and grammar errors

**To Reproduce**

1. From the landing page go to talwa docs by clicking on the "Get Started" button

**Expected behavior**
Use tools like grammarly to check for spelling and grammatical errors and correct them.

**Actual behavior**

**Screenshots**

**Additional details**

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by tinniaru3005**
**What kind of change does this PR introduce?**
Correct grammar & spelling for the internships page

**Issue Number:**

Closes #459 

**Did you add tests for your changes?**
No

**Snapshots/Videos:**

**If relevant, did you update the documentation?**
Yes

**Summary**

This PR corrects the spelling and grammar errors on the internships page

**Does this PR introduce a breaking change?**
No

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-docs/blob/develop/CONTRIBUTING.md)?**

### CODE REVIEW

**Comment by tinniaru3005** [comment]
I am breaking down the PR into smaller parts to avoid merge conflicts. This will also simplify the review process.

**Comment by Cioppolo14** [comment]
@tinniaru3005 Please fix the first comment so that the issue automatically closes. The PR_GUIDELINES.md file has details.

Please also fix the failing tests. 

Please also fill in the template for the pull request as completely as you can. It greatly helps others searching for code changes in future and helps others understand the summarized rationale for the work you have done.

Thank you!

File: docs/internships/gsoc/application-process.md [hunk 1]
```diff
@@ -3,33 +3,33 @@ id: gsoc-application-process
...
 1. a list of deliverables, quantifiable results for the Palisadoes community,
 1. a detailed description / design document,
 1. an approach,
 1. an approximate schedule and
 1. something of a background text.
-1. We also need you to let us know what other commitments you have for the period of GSoC, for example, do you have any exams or a part time job? In other words &#8211; sell yourselves, as if this was an actual job.
+1. We also need you to let us know what other commitments you have for the period of GSoC, for example, do you have any exams or a part-time job? In other words &#8211; sell yourselves, as if this was an actual job.
 
-In drawing up your proposal you should work with your chosen projects community.
+In drawing up your proposal you should work with your chosen project community.
```
Comment (salman-farooqi) [inline docs/internships/gsoc/application-process.md:19]: In drawing up your proposal you should work with your chosen project's community.

Comment (salman-farooqi) [inline docs/internships/gsoc/application-process.md:19]:
```suggestion
In drawing up your proposal you should work with your chosen project's community.
```

File: docs/internships/outreachy/getting-started.md [hunk 2]
```diff
@@ -41,16 +41,16 @@ This file points out some useful tips on where to find our documentation and how
 
 ### 5.  INSTALLATION.md
 
-Installation guide for the projects can be found in this file. Here, the dependencies to installand the versions, essential configurations are outlined step by step for different operating systems inorder for applicants to successfully run the projects locally on their machines.
+The installation guide for the projects can be found in this file. Here, the dependencies to install and the versions, and essential configurations are outlined step by step for different operating systems for applicants to successfully run the projects locally on their machines.
```
Comment (salman-farooqi) [inline docs/internships/outreachy/getting-started.md:44]:
```suggestion
The installation guide for the projects can be found in this file. Here, the dependencies to install the versions and essential configurations are outlined step by step for different operating systems for applicants to successfully run the projects locally on their machines.
```

**Comment by tinniaru3005** [comment]
@Cioppolo14 The issue is about correcting the grammar and spelling for the entire docs website. It involves a lot of files and almost every line. So I wanted to break down it into smaller PRs. This PR is about correcting the internship-related pages. So, I changed the description so that the issue doesn't close. Please let me know if that looks good to you.

**Comment by tinniaru3005** [comment]
Also, I have filled up the template completely. Thanks a lot for pointing that out. :)

**Comment by Cioppolo14** [comment]
@tinniaru3005 Thanks for updating the PR. Please address these items:
-  I'm okay with leaving this PR as one since they're all editoral changes, so I will leave that up to you. 
- We still have some failing tests.  Can you please fix those? 
- Please change the PR so it will automatically close the issue. The PR_GUIDELINES.md file has details.

**Comment by tinniaru3005** [comment]
@Cioppolo14 can you please help me out in understanding why the tests are failing and how I can solve it.

**Comment by Cioppolo14** [comment]
@tinniaru3005 I am looking at it.  Have you confirmed your docusaurus is the same version used on Talawa Docs?

**Review by Cioppolo14 — CHANGES_REQUESTED** [review]
Please let me know if updating Docusaurus fixes the test errors.

**Comment by tinniaru3005** [comment]
How can I find out the version of docusaurus? Can you please guide me like change to which file can fix this

**Comment by tinniaru3005** [comment]
@Cioppolo14 I am trying to see how to solve the issue. If nothing works I will close the PR. I will start reworking and make small commits and draft a PR. I guess that will help me understand better where the problem is coming from. Please let me know if you can think of any other way I can fix the failing tests.

**Comment by noman2002** [comment]
@tinniaru3005 Please fix the conflicting files.

**Comment by tinniaru3005** [comment]
> @tinniaru3005 Please fix the conflicting files.

Sure I will fix them. Can you please help me figure out why the tests are failing? I tried understanding the error messages but couldn't figure out how to resolve them. Otherwise, I am considering closing this PR and opening a new one with smaller commits. That will probably help me better understand how to fix it. Can you please tell me if that's fine or if should I consider some other approach?

**Comment by Cioppolo14** [comment]
@tinniaru3005 Some questions to help:
1) Are you using the yarn.lock file from here: https://github.com/PalisadoesFoundation/talawa-docs/blob/develop/yarn.lock  ? 
2) The conflicting files probably have new versions from other PRs, so please make sure you are using the latest version. 
3) Are you using Docusaurus version 2.3.1, the latest stable version?

**Comment by tinniaru3005** [comment]
Thanks a lot @Cioppolo14, I am looking into it.

**Comment by Cioppolo14** [comment]
@tinniaru3005 Can you resolve the conflicting files?

**Comment by tinniaru3005** [comment]
> @tinniaru3005 Can you resolve the conflicting files?

Sure @Cioppolo14, I am doing it in a few hours. Got a bit busy with writing the proposal. Thanks a lot for your patience and also for reminding me. :)

**Comment by tinniaru3005** [comment]
@Cioppolo14 I have solved the merge conflicts. Thanks again for reminding me on that :)

**Comment by Cioppolo14** [comment]
@tinniaru3005 I'm still seeing two conflicting files, really just ideas.md. Can you address this?

**Comment by tinniaru3005** [comment]
@Cioppolo14 I solved it :)

**Comment by tinniaru3005** [comment]
@Cioppolo14 I tried checking the possibilities to why the tests are failing but couldn't figure it out. The other PR #429  that I raised worked fine, though.

**Comment by Cioppolo14** [comment]
@tinniaru3005 There were some significant changes to the website yesterday, can you address the conflicting files?

**Comment by Cioppolo14** [comment]
We've had significant changes to the page, and we'd like to pause grammar and spelling changes for now, so I am going to close this PR and the related issue.

**Pull request closed without merging**